ajax/libs/image-map-resizer/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "image-map-resizer",
+  "version": "1.0.3",
+  "filename": "imageMapResizer.min.js",
```

Comment:
The filename would be `"js/imageMapResizer.min.js"`, and you should put the files under js folder.